domain/entity/musics.go

```diff
@@ -0,0 +1,8 @@
+package entity
```

Comment:
The main purpose of the domain-driven design is to approach the system from domains' perspective. In other words, what DDD prefers is to collect domain-related codes together, not separating them. However, if there are 'entity', 'repository', and 'service' packages, then we would end up separating domain-related codes. How about making 'musics' package instead, so we could gather the music-related codes together?

Such as:

- domain
  - musics
    - entity
    - repository
    - service
  - user
    - entity
    - repository
  ...

Reply:
I agree.
I think it's a reasonable idea because the package called music can abstract all repository, service, and entity.

Reply:
The current specification of myply is not that complicated, so it doesn't matter, but if the specification becomes more complicated, the area represented by `music` will become ambiguous.

In other words, if an `aggregate` (e.g. containitng music domain or/and user domain or/and video domain) including music is created, when the proposed directory structure is taken, it is obviously difficult to distinguish which domain to put the file created by the developer in.

Reply:
@minkj1992

Maybe it's because I don't have enough experience.

wouldn't this be the same for role-based directory structures?
i think that the question of which directory to put in is the same as the question of which file to put in.